src/cli.rs

```diff
@@ -1,19 +1,14 @@
-use std::path::PathBuf;
-
 use clap::{App, Arg};
 
 use crate::errors::*;
-use crate::filesystem::binding::Binding;
 use crate::filesystem::validation::{binding_validator, path_validator};
 use crate::filesystem::FileSystem;
 
 pub const DEFAULT_ROOTFS: &'static str = "/";
-pub const DEFAULT_CWD: &'static str = ".";
+pub const DEFAULT_CWD: &'static str = "/";
```

Comment:
Can you explain this to me? Why are we changing from `.` to `/`?

Reply:
I changed it from `.` to `/`, because current work dir (`.`) may be outside the new root(`/`), and it is hard to detranslate path `.`.

Furthermore, the default cwd for `chroot` is also `/` instead of `.`.

```sh
sudo chroot / /bin/sh -c "pwd -P"
# The output is `/`
```